helm/dex-app/values.yaml

```diff
@@ -38,7 +38,7 @@ oidc:
       cliendID: ""
       redirectURI: ""
     gsCLIAuth:
-      clientID: ""
+      clientID: "zQiFLUnrTFQwrybYzeY53hWWfhOKWRAU"
```

Comment:
It's set here:
https://github.com/giantswarm/kubectl-gs/blob/a6a953cec7d2d9a7392086cb28934586f2854b06/cmd/login/oidc.go#L24